gammapy/utils/docs.py

```diff
@@ -36,9 +36,7 @@
 
 
 class AccordionHeader(Directive):
-    """
-    Inserts HTML code to open an accordion box in the How To.
-    """
+    """Inserts HTML code to open an accordion box in the How To."""
```

Comment:
pydocstyle ask for imperative wording, so it should be like this:

```suggestion
    """Insert HTML code to open an accordion box in the How To."""
```